script.js

```diff
@@ -0,0 +1,97 @@
+var script = document.createElement('script');
+script.src = "https://checkout.razorpay.com/v1/checkout.js";
```

Comment:
can this be made into another `wp_enque_script` call? That supports external scripts, I think.